spec/support/common_methods.rb

```diff
@@ -12,5 +12,5 @@ def save_xml_in_temp_file(_xml)
     fp.puts "<!-- generated by rspec #{Time.now} -->"
     fp.puts requestor.formatted_xml
   end
-  puts "XML saved in #{filename}"
+  Rails.logger.info "XML saved in #{filename}"
```

Comment:
I wasn't sure if this was necessary - or could be removed - so compromised by sending to logger.